.travis.yml

```diff
@@ -3,7 +3,8 @@ git:
   depth: false
 cache:
   ccache: true
-  directories: $HOME/Library/Caches/Homebrew
+  directories: 
+    - "$HOME/Library/Caches/Homebrew"
```

Comment:
This change is unnecessary, and the previous syntax was simpler. Let's revert this.

Reply:
Shows a warning as bad yml when it's not in list format.